### ISSUE TITLE
Please add ARM64 builds

### ISSUE THREAD

**Issue by inklesspen** [ntrippar/sekey#55]
Obviously M1 macs have Rosetta, but for something important like this, an ARM64 build is a good idea.

**Comment by zachberger** [comment]
This would be dependent on Rust support, which according to rust-lang/rust#73908 is only in beta and is unsupported at this time

**Comment by lean0x2F** [comment]
it's fully supported, I just built it from an m1 without any issues.


```
~/tmp/sekey on  master! ⌚ 1:34:12
$ file target/release/sekey
target/release/sekey: Mach-O 64-bit executable arm64
```

**Comment by tristan-k** [comment]
@lean0x2F Can you publish your build?

**Comment by taxilian** [comment]
For anyone willing to trust an arbitrary stranger on the internet:

https://newcloud.zzt.net/index.php/s/HQm2pRFgieL68dE

Built on my 14" macbook pro M1 max using cargo/rust from macports from the latest master, signed with my own Developer key. If someone has a better place to host it that might be good...

**Comment by marcan** [comment]
@taxilian it looks like that build isn't properly signed with entitlements; checking the entitlements with `codesign` returns nothing.

**Comment by taxilian** [comment]
Yeah, I thought it was working and then discovered I was looking at the wrong thing; haven't had time to figure out why it's not working yet. I followed the instructions, but something didn't happen it seems....

**Comment by taxilian** [comment]
Well, I found one or two definite reasons for it to not work, but something must be missing; if I follow the instructions on the main page to build it ends up with an invalid signature on the app bundle because you add the binary after signing the bundle. If I sign the bundle after copying the binary in then the binary immediately gets killed

```
$ ./bundle/Applications/SeKey.app/Contents/MacOS/sekey -c test
[1]    2084 killed     ./bundle/Applications/SeKey.app/Contents/MacOS/sekey -c test
```

I don't know what else to try

**Comment by taxilian** [comment]
it's frustrating that it builds and (probably) runs just fine, but it's the code signing and packaging that I can't get to work, so it can't use the entitlements and access the secure enclave. any suggestions from @ntrippar would be appreciated =]
